docs/content/references/sui-api/graphql/overview.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Overview
+description: An overview of the Sui GraphQL RPC service.
+---
+
+## About Sui GraphQL RPC
+
+The ability to define precisely the data you want—and only the needed data is a powerful advantage over traditional REST API endpoints. GraphQL lets you replace multiple REST requests with a single call to fetch the required data. Sui provides a GraphQL implementation to serve the data available on-chain, and this page describes how to get started, where to find query examples, and the current limitations.
+
+## Getting Started
+
+The Sui GraphQL RPC service can be accessed in multiple ways. An [online interactive IDE](https://mainnet.sui.io/rpc/graphql) is available to enable users to familiarize themselves with the schema and available queries. 
+If you use TS SDK, the [Guides](./guides.mdx) page will provide a migration guide on how to use the updated TS SDK that leverages the new GraphQL implementation. For developers who used to query the JSON RPC endpoints, they will now have the option to query the new GraphQL enpdpoint in a similar fashion as before (see next section).
+
+
+### High Level Overview
+
+#### Top Level Queries
+In GraphQL data is retrieved through queries in a JSON format. At the top level, there are a number of [queries](./reference/queries/address) that can be executed.
+
+#### Connections
+
```

Comment:
Just a note that these pages are reachable through search. Okay with having blank sections if someone lands here? If not, remove and put back when ready.

Reply:
@ronny-mysten searchable through google search or through our GitHub?

Reply:
Through the docs.sui.io site search.